src/server/services/toerisme.ts

```diff
@@ -0,0 +1,37 @@
+import { requestData } from '../helpers';
+import { getApiConfig } from '../config';
+import { GenericDocument } from '../../universal/types/App.types';
+
+export interface ServiceNameSource {
+  title: string;
+  documents: GenericDocument[];
+}
+
+// The root type of the api source
+export type ServiceNameSourceData = ServiceNameSource[];
+
+// The items you want to output in this api
+export interface Toerisme extends ServiceNameSource {}
+
+// The root type of the items you want to output in this api
+export type ToerismeData = Toerisme[];
+
+// The function you can use to transform the api source data
+export function transformToerismeData(
+  responseData: ServiceNameSourceData
+): ToerismeData {
+  return responseData;
```

Comment:
Omdat we van de Api een response krijgen met een content key `{ content: [] }` en de response in de BFF óók in een `{ content: responseData }` gewrapped wordt, krijgen we `{ content: { content: [ ... ] } }` structuur. De transform functie moet dus iets doen als; `return responseData.content`.